backend/api/database/entities/model.py

```diff
@@ -0,0 +1,66 @@
+"""
+Declare the model objects for the database as well as schema for each
+"""
+from marshmallow import Schema, fields
+from sqlalchemy import Column, String, ForeignKey
+from sqlalchemy.orm import relationship
+
+from api.database.entities.entity import Entity, BASE
+from api.database.util import url_safe_uuid
+
+
+class User(Entity, BASE):
+    """
+    The user entity.
+    """
+    __tablename__ = 'users'
+
+    user_id = Column(String, primary_key=True)
+    servers = relationship("Server", cascade="all, delete-orphan")
+
+    def __init__(self, user_id):
+        Entity.__init__(self)
+        self.user_id = user_id
+
+
+class UserSchema(Schema):
+    """
+    The users schema.
+    """
+    user_id = fields.Str()
+    created_at = fields.DateTime()
+    updated_at = fields.DateTime()
+
+
+class Server(Entity, BASE):
+    """
+    The server entity.
+    """
+    __tablename__ = 'servers'
+
+    server_id = Column(String, primary_key=True)
+    user_id = Column(String, ForeignKey('users.user_id'))

```

Comment:
Is this the 'owner' ? And can a server thus have multiple users but one owner?

Otherwise why would you link a user to servers and a server to a user?